packages/@react-aria/listbox/src/useOption.ts

```diff
@@ -26,32 +26,59 @@ interface OptionAria {
   labelProps: HTMLAttributes<HTMLElement>,
 
   /** Props for the description text element inside the option, if any. */
-  descriptionProps: HTMLAttributes<HTMLElement>
+  descriptionProps: HTMLAttributes<HTMLElement>,
+
+  /** Whether the option is currently focused. */
+  isFocused: boolean,
+  /** Whether the option is currently selected. */
+  isSelected: boolean,
+  /** Whether the option is currently in a pressed state. */
+  isPressed: boolean,
+  /** Whether the option is disabled. */
+  isDisabled: boolean
 }
 
 interface AriaOptionProps {
-  /** Whether the option is disabled. */
+  /**
+   * Whether the option is disabled.
+   * @deprecated
```

Comment:
Deprecating these options as they are now either computed from the passed in state, or should be passed to the listbox instead.

Reply:
should we still display these in our docs for a little bit? looks like currently they don't